src/Core/Logger/Type/StreamLogger.php

```diff
@@ -92,7 +92,7 @@ public function __construct(string $channel, IManageConfigValues $config, IHaveC
 		$this->fileSystem = $fileSystem;
 
 		$stream = $this->logfile ?? $config->get('system', 'logfile');
-		if ((file_exists($stream) && !is_writable($stream)) || is_writable(basename($stream))) {
+		if ((@file_exists($stream) && !@is_writable($stream)) || @is_writable(basename($stream))) {
```

Comment:
Why do we throw an Exception when the base name is writable?

Reply:
because the `vfs://` prefix of the file isn't supported by some of the PHP native functions, like `file_exists` and, AFAIK, `is_writable` as well. It doesn't break the tests, but throws `E_WARNING`s all over the output.

Reply:
I’m not asking about the eroor suppression operator, I’m asking about the whole condition. An exception is thrown if the filename exists and is *not* writable or if the base folder *is* writable?

Reply:
got it .. wrong boolean logic *g*